app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -91,8 +108,8 @@
                           class="subTitleObjectStyle"
                           style="font-size: 14px !important"
                         >
-                          {{ selected.length }} of
-                          {{ FILTER_HEADERS.length }} selected for export
+                          {{ selected.length }} of {{ FILTER_HEADERS.length }}
```

Comment:
The `of` here isn't being translated. Maybe replace it with a slash? `X/Y selected for export`